sdk/storage/Azure.Storage.Blobs.Batch/src/BatchConstants.cs

```diff
@@ -25,6 +25,8 @@ internal static class BatchConstants
         public const string BatchSeparator = "--";
         public const string HttpVersion = "HTTP/1.1";
 
+#pragma warning disable CA1802 // Use literals where appropriate
```

Comment:
This wasn't failing in pipelines, do you have more sensitive settings locally or newer dotnet?
If possible I'd rather address these warning. If we silence them then we'll loose opportunity to fix them later.
Otherwise I'd remove these from PR.

I've seen couple of these in PR, please take a look at them as well.

Reply:
I'm assuming it is a newer dotnet. The reason I disabled the warning is that we'd have to switch this to a constant rather than an interpolated string using the `nameof` method. If you prefer changing to a constant, I'm happy to make that change. Otherwise, I'd leave this until interpolated constant strings comes out of preview support.

Reply:
gotcha, lets leave it if this is upcoming feature.